test_teardown.stress_ng.yml

```diff
@@ -0,0 +1,17 @@
+---
+# Removal of stress-ng tool and all it's dependencies (checked via yum.log),
```

Comment:
To be slightly more precise - I would mention here, that we are removing only dependencies from EPEL repository (it is partially mentioned on the next line, but still I would slightly rephrase it to be clear).

Reply:
Good catch, will fix this.